CONTRIBUTING.txt

```diff
@@ -394,6 +394,29 @@ can be installed using: ``scikit-image/requirements/docs.txt`` file.
 
     pip install -r requirements/docs.txt
 
+.. _sphinx_gallery:
+
+**Sphinx-Gallery:**
+
+The above-mentioned command includes the installation of Sphinx-Gallery.
+The gallery of :ref:`examples_gallery`
+is created using `Sphinx-Gallery <https://sphinx-gallery.github.io>`_.
```

Comment:
```suggestion
The above install command includes the installation of
`Sphinx-Gallery <https://sphinx-gallery.github.io>`_, which we use to create
the :ref:`examples_gallery`.
```

Reply:
Since
https://github.com/scikit-image/scikit-image/blob/c82a3c8c9d6187a056a43c7b7ccae26f6111cf93/doc/examples/README.txt#L1-L4
the sentence will read as "... to create the General examples" -- I'm fine with it.